sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/validation/ZoneValidator.java

```diff
@@ -20,52 +20,44 @@
     public static Collection<ConfigValidationError> validate(
             Optional<ServiceSpec> oldConfig,
             ServiceSpec newConfig,
-            String podType,
-            String taskName) {
+            String podType) {
```

Comment:
Could simplify this for callers with a `String... podTypes` where all passed podTypes are checked. Then e.g. HDFS could call it as `validate(..., NAME_POD_TYPE, JOURNAL_POD_TYPE, DATA_POD_TYPE)`